src/dev/flang/util/unicode/ParseUnicodeData.java

```diff
@@ -233,6 +263,13 @@ boolean isLast()
       {
         finishBlock();
       }
+  }
+
+
+  private void PrintJava()
```

Comment:
please add JavaDoc comment. Java naming conventions require method names to start with lower case.